.github/actions/is-pr-approved/src/main.ts

```diff
@@ -0,0 +1,161 @@
+import * as core from '@actions/core'
+import {Context} from '@actions/github/lib/context'
+import {GitHub} from '@actions/github/lib/utils'
+import {actionInputs, transformIfSet} from 'github-actions-utils'
+
+interface GitHubAPIError {
+  status: number
+  request: any // eslint-disable-line @typescript-eslint/no-explicit-any
+  response: any // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+export const DEFAULT_ALLOWED_ACTORS: readonly string[] = [
```

Comment:
NIT:
What do you think using an input parameter with this list of BOT actors rather than being hardcoded? This can help to be consumed this action by others and configure those actors if needed.

Reply:
It's possible to override the list of allowed actors using the `allowed-actors` input.
However, Hardcoding default allowed actors in the code rather than in the `actions.yml` for test purposes is easier.

Reply:
The default list of allowed actors is present in the `How to use it` section.